presto-main/src/main/java/com/facebook/presto/operator/scalar/JsonPathTokenizer.java

```diff
@@ -35,7 +34,11 @@
 
     public JsonPathTokenizer(String path)
     {
-        this.path = checkNotNull(path, "path is null");
```

Comment:
This should stay `checkNotNull` since this is an internal class that won't be called directly. Passing null here would an error in the JSON functions (thus not a user error).